### PR TITLE
Flowblade update to 2.16.3

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)

### CODE REVIEW

**Comment by zlice** [comment]
saw https://github.com/jliljebl/flowblade/issues/329#issuecomment-1085142449 mention `SDL_VIDEOOVERRIDE` ? should that be added with the gdk var? i dont use flowblade and ive barely messed with wayland still because of these types of things

**Comment by Luciogi** [comment]
> saw [jliljebl/flowblade#329 (comment)](https://github.com/jliljebl/flowblade/issues/329#issuecomment-1085142449) mention `SDL_VIDEOOVERRIDE` ? should that be added with the gdk var? i dont use flowblade and ive barely messed with wayland still because of these types of things

GDK var application ui is working

File: srcpkgs/flowblade/INSTALL.msg
Comment (classabbyamp) [inline srcpkgs/flowblade/INSTALL.msg:1]: remove this

File: srcpkgs/flowblade/files/enforce-x11.sh
Comment (classabbyamp) [inline srcpkgs/flowblade/files/enforce-x11.sh:1]: just call this `files/flowblade-wayland`

File: srcpkgs/flowblade/files/enforce-x11.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Flowblade does not support wayland and it crashes
+GDK_BACKEND=x11 /usr/bin/flowblade $@
```
Comment (classabbyamp) [inline srcpkgs/flowblade/files/enforce-x11.sh:4]:
```suggestion
exec env GDK_BACKEND=x11 /usr/bin/flowblade "$@"
```

File: srcpkgs/flowblade/template [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+
+do_patch() {
+	# Desktop file should run /usr/bin/flowblade.sh
+	vsed -i installdata/io.github.jliljebl.Flowblade.desktop -e 's#env.*#/usr/bin/flowblade.sh %f#'
+}
+
+post_install() {
+	# Enforce X11, otherwise it crashes on wayland
+	vbin "${FILESDIR}/enforce-x11.sh" flowblade.sh
+}
```
Comment (classabbyamp) [inline srcpkgs/flowblade/template:28]:
```suggestion
post_install() {
	vbin "${FILESDIR}/flowblade-wayland"
}
```

the desktop file already sets the env var, so the script should only be necessary for running from cmdline: https://github.com/jliljebl/flowblade/blob/v2.14.0.2/flowblade-trunk/installdata/io.github.jliljebl.Flowblade.desktop#L6

File: srcpkgs/flowblade/template [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
+version=2.14.0.2
 revision=1
 build_wrksrc=flowblade-trunk
 build_style=python3-module
 make_install_args="--install-lib=${DESTDIR}/usr/share/flowblade"
 hostmakedepends="python3-setuptools"
 # https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/DEPENDENCIES.md
-depends="frei0r-plugins gmic mlt-python3 python3-Pillow python3-dbus gtk+3
- python3-gobject python3-numpy swh-plugins"
+depends="frei0r-plugins gmic mlt7 mlt7-python3 python3-cairo python3-Pillow python3-gobject python3-numpy gtk+3
```
Comment (classabbyamp) [inline srcpkgs/flowblade/template:10]:
```suggestion
depends="frei0r-plugins gmic mlt7-python3 python3-cairo python3-Pillow python3-gobject python3-numpy gtk+3
```
mlt7-python3 depends on mlt7

File: srcpkgs/flowblade/template [hunk 4]
```diff
@@ -1,17 +1,28 @@
 # Template file for 'flowblade'
 pkgname=flowblade
-version=2.8.0.3
+version=2.14.0.2
```
Comment (classabbyamp) [inline srcpkgs/flowblade/template:3]: 2.16.3 available

**Pull request merged**